charts/prometheus-redis-exporter/templates/extra-manifests.yaml

```diff
@@ -0,0 +1,4 @@
+{{ range .Values.extraManifests }}
+---
+{{ tpl (toYaml .) $ }}
```

Comment:
Can you show some tests to prove this work?

Reply:
I've added a ci directory to add tests